src/main/java/com/dev/cinema/model/User.java

```diff
@@ -16,6 +17,16 @@
     @Column (unique = true)
     private String email;
     private String password;
+    @ManyToOne
+    private Role role;
```

Comment:
user can have a lot of roles, so let's use Set of roles here

Reply:
done